app/lib/features/activities/widgets/notification_card.dart

```diff
@@ -0,0 +1,149 @@
+import 'package:acter/common/providers/common_providers.dart';
```

Comment:
I just assume this is only a copy-paste.

Reply:
That file uses `chatProfileDataProvider` twice and `chatProfileDataProvider` is defined in `common_providers.dart`.